### PR TITLE
Support multi labels textcat pipe for transformers

### LINKED ISSUE #9: support multi-label classification
Is camphr expected to support multi-label classification?  If so, let me try to implement it.
I will send PR within a few days.

### PR DESCRIPTION
**Pull request by tamuhey**
closes #9

- Add `TrfForMultiLabelSequenceClassification` for multiple text classification.
    -  pipe name: `transformers_multilabel_sequence_classifier`
- Add docs for fine-tuning multi textcat pipe
    - https://github.com/PKSHATechnology-Research/camphr/blob/feature%2Fmulti-textcat/docs/source/notes/finetune_transformers.rst#multilabel-text-classification

### CODE REVIEW

File: camphr/pipelines/transformers/seq_classification.py [hunk 1]
```diff
@@ -73,6 +65,15 @@ class TrfForSequenceClassification(
 
     model_cls = TrfSequenceClassifier
 
+    def set_annotations(self, docs: Iterable[Doc], logits: torch.Tensor):
+        probs = torch.softmax(logits, 1)
```
Comment (TKSTo) [inline camphr/pipelines/transformers/seq_classification.py]: For multi-label classification, softmax should not be called. 
(Sorry for no suggestion code)

**Comment by TKSTo** [comment]
How about implementing _labels method, mainly for multi-label classification?
The method name is up to you. `_labels` may be ambiguous. 
```
def _labels(doc: Doc) -> List[Tuple[str, Any]]:
    if not doc.cats:
        return []
    return list(doc.cats.items())
```

**Comment by tamuhey** [comment]
It's not a very good way.
Since `doc.cats` contains all labels, it is same as using `doc.cats` directly.
You need to set a threshold to return high scoring labels, but I think this should be left to user.

**Comment by TKSTo** [comment]
> Since doc.cats contains all labels, it is same as using doc.cats directly.

I agree with you! I'm sorry!

**Comment by TKSTo** [comment]
Is it possible to validate ".jsonl" for textcat (not multitextcat)?
To avoid the below case.

When trying multitextcat and forgetting to switch the argument from model.textcat_label to model.multitextcat_label, the training phase works as textcat(not multitextcat) even if the sum of labels is not 1.

**Comment by tamuhey** [comment]
Added.
